app/views/step_by_step_pages/publish_or_delete.html.erb

```diff
@@ -47,7 +38,14 @@
                 destructive: true
               } %>
             <% end %>
-          <% else %>
+          <% end %>
+          <% if @step_by_step_page.can_be_published? %>
```

Comment:
Worth making this an `elsif` instead?

Reply:
I don't think it makes much of a difference.

Reply:
I've added an `elsif` to handle showing the error message.